src/managers/transaction-parser.ts

```diff
@@ -0,0 +1,34 @@
+import * as ynab from "ynab";
+import {FullTransaction} from "./amazon-manager";
+
+
+export const parseTransactions = (options: {
```

Comment:
I think this can be `export function`?

Reply:
These are arrow functions - you can read up on the differences but in this case its just stylistic and a more concise syntax.

Reply:
I would say it's confusing to define functions as constants, I see no downsides to using the standard `function` syntax, and I don't see how it's more concise? The alternative would be

`export function parseTransactions(options: {....}) { `
which is shorter.